### PR TITLE
feat(i18n): add Euskara/Basque localisation

### PR DESCRIPTION
**Pull request by xabirequejo**
Not sure this is the right way. I have tried Codeflow like your contributing documents suggests but I'm too new to this.
Anyway, I have included `eu` in `config/i18n.ts` and I have also created a `eu.json` file in `locales`.
en-US is empty. Am I looking at the right file?
Let me know if I need to do something else/differently.
Thanks

### CODE REVIEW

File: config/i18n.ts [hunk 1]
```diff
@@ -104,6 +104,11 @@ const locales: LocaleObjectData[] = [
     file: 'es.json',
     name: 'Español',
   },
+  {
+    code: 'eu',
+    file: 'eu.json',
```
Comment (userquin) [inline config/i18n.ts:109]: add the country, `eu-ES` for code and move `eu.json` to `eu-ES.json`

File: locales/eu.json [hunk 2]
```diff
@@ -0,0 +1,593 @@
...
+    },
+    "delete_posts": {
+      "cancel": "Utzi",
+      "confirm": "Ezabatu",
+      "title": "Ziur bidalketa hau ezabatu nahi duzula？"
+    },
+    "mute_account": {
+      "cancel": "Utzi",
+      "confirm": "Mututu",
+      "title": "Ziur {0} mututu nahi duzula？"
```
Comment (userquin) [inline locales/eu.json:135]: can you review the question mark here and on L130? check the question mark in line 140

Comment (userquin) [reply locales/eu.json:135]: also on lines 111, 116 and 125

**Review by userquin — APPROVED** [review]

**Comment by userquin** [comment]
@patak-dev another official language in Spain: Basque (Euskara)

**Comment by userquin** [comment]
@xabirequejo Gracias Xabi

**Review by patak-dev — APPROVED** [review]
Gracias!

**Pull request merged**